sources/packages/web/src/assets/css/base.scss

```diff
@@ -203,3 +189,11 @@ html * {
 .v-text-field-search-width {
   width: 305px;
 }
+
+.success-chip-background {
+  background-color: $banner-background-success !important;
+}
+
+.warning-chip-background {
```

Comment:
Are we planning to add the other ones as needed?

Reply:
as per the design system, badge with background color only these 2, (success and warning, i have added them both)
![image](https://user-images.githubusercontent.com/77353155/182655860-325207f1-2ff4-4f10-862f-06b2e041e30d.png)

Reply:
Let's talk about it. I believe that maybe we can have these components organized in a different way.